app/routes/screens/details/Details.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 
 import { View, Text, Image } from "react-native";
-import { Icon } from "react-native-elements";
+import { Icon, Card } from "react-native-elements";
```

Comment:
yep, we should definitely do that. 
Only few components from that lib are still used....